CoreWidgetProvider/CoreWidgetProvider.csproj

```diff
@@ -41,6 +41,9 @@
     <Content Include="Widgets\Templates\SSHWalletTemplate.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+	  <Content Include="Widgets\Templates\SSHWalletPatternMatching.json">
```

Comment:
Please fix formatting to match above lines.

Reply:
Fixed.

Reply:
ok phew - I've fixed it.